examples/model_selection/plot_det.py

```diff
@@ -117,3 +122,35 @@
 # DET curves give direct feedback of the detection error tradeoff to aid in
 # operating point analysis. The user can then decide the FNR they are willing to
 # accept at the expense of the FPR (or vice-versa).
+#
+# Chance level of the ROC and DET curves
+# --------------------------------------
+#
+# Chance level is in general not easy to define, as it depends on what is the
```

Comment:
> Chance level is in general not easy to define

This is precisely why we should avoid speaking about chance level in this paragraph, in my opinion. Let's just mention "Non-informative classifiers" instead: they are easy to define: those are any classifiers that make a prediction of class membership that is independent of the values of the features passed as input `X` to the `predict_proba` or `decision_function` methods.

Let's stop using ambiguous / misleading terms in our doc if have perfectly valid alternatives.